manage

```diff
@@ -20,7 +20,9 @@ if [ -z "$1" ];then
 fi
 
 set -a #make all functions in the api available to apps
-source "${DIRECTORY}/api" || error "failed to source ${DIRECTORY}/api"
+if [ "$1" != refresh ] || ! command -v generate_logo >/dev/null ;then
```

Comment:
Wouldn't it be better to check for `status` or `status_green` rather than `generate_logo`, a function that isn't used by `refresh`? Not that it really makes any difference

Reply:
I wanted something that would have no chance of an overlap with an actual `status` command in /usr/bin.

Reply:
thats why you use `typeset -f status &>/dev/null` and not `command -v`

Reply:
resolved with https://github.com/Botspot/pi-apps/pull/2340/commits/29e05be3979c3203ea30d480577af23400d8cc06 no longer needed